openjdk8/src/main/docker/Dockerfile

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google_appengine/base
+FROM ${docker.appengine.image}
```

Comment:
For CI we will want to build openjdk-runtime, jetty-runtime and jetty-gae-compat runtime in a series, but we may also have other CI builds of branches and alternatives, thus their is the risk of collision in the docker namespace.   Also eventually the default values for the jetty-runtime and jetty-gae-compat will be to pull released images from gcr.io.

So I parametrized, so properties can be used to control the docker name space and link up the images in CI build chains, without needing to release into gcr.io etc.

I guess it is not really needed at this bottom layer... but this keeps the pattern and does allow for other base images to be easily injected.